src/movement/test/control_system.test

```diff
@@ -0,0 +1,14 @@
+<launch>
+    <test test-name="test_control_system" pkg="robosub" type="test_control_system" />
+
+    <node name="control" pkg="robosub" type="control"/>
+
+    <rosparam command="load" file="$(find robosub)/param/cobalt.yaml" />
+
+    <include file="$(find robosub_simulator)/launch/gazebo.launch">
```

Comment:
It is required, the control system relies on the simulator to simulate the physics of the submarine.